corehq/apps/smsbillables/models.py

```diff
@@ -365,3 +376,39 @@ def _get_usage_fee(cls, domain, direction):
                 % (direction, domain)
             )
         return usage_fee
+
+    @classmethod
+    def _get_twilio_charges(cls, backend_instance, backend_message_id, direction, couch_id):
+        def _get_twilio_client(twilio_backend_id):
+            twilio_backend = SQLMobileBackend.load(
+                twilio_backend_id,
+                api_id=SQLTwilioBackend.get_api_id(),
+                is_couch_id=True,
+                include_deleted=True,
+            )
+            config = twilio_backend.config
+            return TwilioRestClient(config.account_sid, config.auth_token)
```

Comment:
should only charge global backends: https://github.com/dimagi/commcare-hq/pull/10220/files#diff-f0b5e40a6b2df9c55372280d3e0c8a19R336